test-in-docker

```diff
@@ -6,6 +6,7 @@ set -eu
 default_version='4.3.11'
```

Comment:
Maybe we should set the minimum version to something like 5.3 in regard of #877

Reply:
I started to do that in a94df3ddcb027d88717f854cd5f610a771c1c71c but then remembered why I have it the way I do.

The default should be the _oldest_ version of zsh we support, not newest.

Either way, it is easier to change now and it has the same lookup properties the command line does.

Reply:
Well. The oldest supported version of ZSH _is_ 5.3 in `next` (at least in the battery segment). Otherwise it is [5.1](https://github.com/bhilburn/powerlevel9k/pull/859#issuecomment-397842819).